cppcache/src/CMakeLists.txt

```diff
@@ -92,6 +92,7 @@ endif()
 
 target_link_libraries(_apache-geode INTERFACE
   ACE
+  boost
```

Comment:
Are you sure this is absolutely necessary right now? I think the exception bits are header only.

Reply:
I think you're probably right.  I'll see if I can just add boost headers to include path.

Reply:
You may have to add a project dependency to make sure Boost is “built” before we build.